src/main/java/Task.java

```diff
@@ -1,6 +1,6 @@
 public class Task {
 
-    /** The actual description or name of the task */
+S    /** The actual description or name of the task */
     private String description;
```

Comment:
I'm not too sure but if the description stays the same, would setting it as final help in abstraction/encapsulation?